src/sentry/testutils/cases.py

```diff
@@ -971,6 +971,10 @@ def store_session(self, session):
         if status != "ok":  # terminal
             self._push_metric(session, "distribution", "session.duration", {}, session["duration"])
 
+    def bulk_store_sessions(self, sessions):
+        for session in sessions:
```

Comment:
Nit: `self.store(session) for session in sessions`

Reply:
Not sure what are you suggesting ?
are you suggesting:

```
def bulk_store_sessions(self, sessions):
         [self.store_session(session) for session in sessions ]
```

That would generate a list for no good reason

or:

```
def bulk_store_sessions(self, sessions):
         (self.store_session(session) for session in sessions)
```

that would create a generator that would not do anything.

Reply:
Oh nvm, ignore this plz.. misread, thought you were returning a list